wps-cloudformation-template.yaml

```diff
@@ -585,6 +585,81 @@ Resources:
                 Effect: "Allow"
                 Action: ["ses:SendRawEmail"]
                 Resource: ["*"]
+  LambdaClearBucketFunction:
+    Type: "AWS::Lambda::Function"
+    Condition: CreateEphemeralBucket
+    Properties:
+      Code:
+        ZipFile: |
+          import json
+          import boto3
+          import os
+          import cfnresponse
+
+          def handler(event, context):
+               print event
+               print context
+               try:
+
+                 if event['RequestType'] == 'Delete':
+                   s3 = boto3.resource('s3')
+                   bucket = s3.Bucket(os.getenv('BUCKET'))
+                   for obj in bucket.objects.all():
+                     obj.delete()
+                 cfnresponse.send(event, context, cfnresponse.SUCCESS, {'Data': 'OK'}, 'CustomResourcePhysicalID')
```

Comment:
Maybe we should use more meaningful _CustomResourcePhysicalID_ and make sure it is different for different resources.